css/style.css

```diff
@@ -9,7 +9,7 @@
 	position: absolute;
 	top: 0;
 	width: 100%;
-	z-index: 1000;
+	z-index: 3000;
```

Comment:
@jancborchardt could you please add a css comment why the z-index has to be exactly 3000.